src/main/kotlin/com/ekino/oss/plugin/KotlinQualityPlugin.kt

```diff
@@ -82,7 +87,10 @@ class KotlinQualityPlugin : Plugin<Project> {
       )
 
       configure<DetektExtension> {
-        config = files(configTempFile)
+        config = files(
+          configTempFile,
+          extensions.getByType(KotlinQualityPluginExtension::class.java).customDetektConfig
```

Comment:
maybe cleaner with 
```
          import org.gradle.kotlin.dsl.getByType
          ...
          extensions.getByType<KotlinQualityPluginExtension>().customDetektConfig
```

Reply:
Thanks for the review. I agree.

It's done :)